### PR TITLE
Support for dragging multiple character/outfit cards at the same time in studio

### PR DESCRIPTION
**Pull request by Timo713**
This PR is related to this issue: https://github.com/IllusionMods/DragAndDrop/issues/12

Thanks to the power of AI, I managed to edit this plugin to allow for multiple card loading into KK studio slots, it adds a new method for loading all the selected cards in a folder whenever they're dragged over the selected slots in studio. I've checked and it hasn't broken any other functionalities yet, but you're free to review the code and edit it as it might need some polishing.

I tried to add the option to load them into the classroom screen slots too but there's no handler for it yet and without any references on how that screen works I have no idea on what to work with.

### CODE REVIEW

**Review by ManlyMarco — COMMENTED** [review]
Thanks for the PR! Did you build and test these changes in game? I'm pretty sure this won't compile and will have issues outside of studio since it will load each character card onto the same character.

**Comment by Timo713** [comment]
Yes, I've tested it already in maker, free h and live and and no problems so far. Outside of studio, you're only meant to load a character at a time, so I don't see where this could be an issue. I'll include it in case you want to test it too.
[DragAndDropMultipleCardLoading.zip](https://github.com/user-attachments/files/15753612/DragAndDropMultipleCardLoading.zip)

File: src/DragAndDrop.Core/CardHandlerMethods.cs [hunk 1]
```diff
@@ -7,8 +8,9 @@ internal abstract class CardHandlerMethods : CardHandlerCore<CardHandlerMethods>
         public virtual void Scene_Load(string path, POINT pos) { }
         public virtual void Scene_Import(string path, POINT pos) { }
         public virtual void Character_Load(string path, POINT pos, byte sex) { }
+        public virtual void Character_LoadMultiple(List<string> paths, POINT pos, byte sex) { }
```
Comment (ManlyMarco) [inline src/DragAndDrop.Core/CardHandlerMethods.cs]: I wouldn't add this to the API since it's only useful in one specific case.

**Review by ManlyMarco — COMMENTED** [review]
Sorry I went silent about this PR, it works but I didn't merge it yet because it only works for KK and not other games, but that's partially because the overall structure of this plugin isn't ideal. If you could make it also at least work in KKS (ideally also AI and HS2) I would merge it.

**Comment by Timo713** [comment]
I guess I managed to make it work on the other games too, I tested on all of them and no problems so far. The only game I didn't test was PH and I didn't add compatibility for EC since there's no handler for anything that this is useful for.

I also took the liberty to implement the same feature for coordinate loading, so, now you can grab multiple coordinate cards into character slots and each of them will all be loaded separately. Great for previewing lots of clothing and characters!

**Review by ManlyMarco — CHANGES_REQUESTED** [review]
Looks like you forgot to update StudioHandler for Sunshine, build fails. Looks good otherwise.

**Comment by Timo713** [comment]
Woops! sorry! it should be fixed now.

**Review by ManlyMarco — APPROVED** [review]
It works, thank you for the PR!

**Pull request merged**